cocos/tween/actions/action-interval.ts

```diff
@@ -454,25 +454,27 @@ export class Sequence extends ActionInterval {
  * const seq = sequence(actArray);
  */
 // todo: It should be use new
-export function sequence (/* Multiple Arguments */tempArray: any): ActionInterval {
+export function sequence (/* Multiple Arguments */tempArray: any): ActionInterval | null {
```

Comment:
It is better no change the return type. It may break comapatibility.

Reply:
It may not work, because the result may be null. eslint will report an error.

The original meaning of this function was to not allow error conditions, and if an error condition occurs, it will be printed out using the logID

Reply:
How about just return a new ActionInterval without any initialization instead of null?

Reply:
It feels like this doesn't identify if it's reasonable or illegal. If it's illegal, it's also used as a legal value

Reply:
The original implementation also return an uninitialized object.

Reply:
Yeah, it feels reasonable to return null.

Reply:
If there is an error, I think it makes sense to return null to let the user know directly. This doesn't break compatibility if there is no error.

Reply:
You can see the CI, it break compatibility. The game project may meet the same problem.

Reply:
Or maybe we should revert the modification. Just use any as it is. Or it will create an useless object, which will add gc overhead.

Reply:
ok